src/OpenTelemetry.Exporter.Console/ConsoleLogRecordExporter.cs

```diff
@@ -76,13 +80,13 @@ public override ExportResult Export(in Batch<LogRecord> batch)
                 if (logRecord.EventId != default)
                 {
                     this.WriteLine($"{"LogRecord.EventId:",-RightPaddingLength}{logRecord.EventId.Id}");
-                    if (string.IsNullOrEmpty(logRecord.EventId.Name))
+                    if (!string.IsNullOrEmpty(logRecord.EventId.Name))
                     {
                         this.WriteLine($"{"LogRecord.EventName:",-RightPaddingLength}{logRecord.EventId.Name}");
                     }
                 }
 
-                if (logRecord.Exception is { })
+                if (logRecord.Exception != null)
```

Comment:
All the other checks in here were using `!= null` so I changed this just to be consistent.